include/evmc/evmc.h

```diff
@@ -116,6 +118,13 @@ struct evmc_message
      */
     struct evmc_uint256be code_hash;
 
+    /**
+     * The optional value used in new contract address construction.
+     *
+     *  Ignored unless kind == EVMC_CREATE2.
+     */
+    struct evmc_uint256be salt;
```

Comment:
How about we name it `create_salt`?

Reply:
`create2_salt` for explicitlness?

Reply:
Otherwise `creation_salt`?

Reply:
`create2_salt` :+1: 
Similarly to `create_address`.